autoload/fern/scheme/file/util.vim

```diff
@@ -37,7 +37,8 @@ if exists('*readdir')
     let l:Profile = fern#profile#start('fern#scheme#file#util#list_entries_readdir')
     let s = s:is_windows ? '\' : '/'
     let p = a:path[-1:] ==# s ? a:path : (a:path . s)
-    return s:Promise.resolve(readdir(a:path))
+    silent! let children = readdir(a:path)
```

Comment:
:memo:

`readdir` returns an empty list on error.

Reply:
`readdir` returns an empty list on error, **AND** generate an exception.
If it called in function with `abort`, The function exits with an exception before the assignment.